artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -0,0 +1,786 @@
+
+paths:
+  /subscriptions:
+    post:
+      tags:
+        - apiName subscription
+      summary: 'Create a apiName event subscription'
+      description: Create a apiName event subscription
+      operationId: createApiNameSubscription
+      parameters:
+        - $ref: '#/components/parameters/x-correlator'
+      security:
+        - openId:
+          - api-name:event-type:grant-level
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SubscriptionRequest'
+        required: true
+      callbacks:
+        notifications:
+          "{$request.body#/sink}":
+            post:
+              tags:
+                - notifications callback
+              summary: "notifications callback"
+              description: |
+                Important: this endpoint is to be implemented by the API consumer.
+                The apiName server will call this endpoint whenever a apiName event occurs.
+                `operationId` value will have to be replaced accordingly with WG API specific semantic
+              operationId: postNotification
+              parameters:
+                - $ref: '#/components/parameters/x-correlator'
+              requestBody:
+                required: true
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/CloudEvent"
+ 
+              responses:
+                "204":
+                  description: Successful notification
+                  headers:
+                    x-correlator:
+                      $ref: '#/components/headers/x-correlator'
+                "400":
+                  $ref: "#/components/responses/Generic400"
+                "401":
+                  $ref: "#/components/responses/Generic401"
+                "403":
+                  $ref: "#/components/responses/Generic403"
+                "500":
+                  $ref: "#/components/responses/Generic500"
+                "503":
+                  $ref: "#/components/responses/Generic503"
+              security:
+                - { }
+
+      responses:
+        "201":
+          description: Created
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Subscription'
+        "202":
+          description: Request accepted to be processed. It applies for async creation process.
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SubscriptionAsync'
+        "400":
+          $ref: '#/components/responses/CreateSubscriptionBadRequest400'
+        "401":
+          $ref: '#/components/responses/Generic401'
+        "403":
```

Comment:
I think we can take advantage of the output of Issue#163 discussion - https://github.com/camaraproject/Commonalities/issues/163#issuecomment-2022672127

and consider the exception SUSBCRIPTION_MISMATCH for the case the request body does not match the granted scope